src/miner.cpp

```diff
@@ -193,32 +193,21 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     coinbaseTx.vout.resize(1);
     coinbaseTx.vout[0].scriptPubKey = scriptPubKeyIn;
 
-    // Track sidechain tx fees
-    CAmount nSideFees = 0;
```

Comment:
With the removal of the nSideFees where does the sidechain withdrawal fee get set?

Reply:
The mainchain miner's half of the WT^'s fees actually have to be hardcoded by the sidechain when the WT^ is created. When ```CTransaction BlockAssembler::CreateWTPrimePayout(uint8_t nSidechain)``` is called on the mainchain, one of the outputs will be an anyone-can-spend output that the miner can claim (probably with their custom get block template mining servers)